demo/x-app.html

```diff
@@ -16,19 +16,23 @@
 
     :host {
       @apply(--layout-horizontal);
+      @apply(--paper-font-body2);
     }
 
     [nav] {
-      width: 300px;
       @apply(--layout-vertical);
-      background-color: #ddd;
+
+      width: 300px;
```

Comment:
This is the main navigation, and it's usually of fixed width. However, I reduce the width to 200px.